src/commonComponents/cart/Cart.js

```diff
@@ -0,0 +1,75 @@
+// prop-types
```

Comment:
i the preview if i click on any links for example momos then that heading is getting hidden behind subheader
try moving slightly down